packages/lesswrong/components/localGroups/LocalGroupPage.tsx

```diff
@@ -139,6 +157,14 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
           Past Events
         </Components.Typography>
         <PostsList2 terms={{view: 'pastEvents', groupId: groupId}} />
+        
+        {group.googleLocation && <CommunityMapWrapper
+            className={classes.mapContainer}
+            terms={{view: "events", groupId: groupId}}
+            groupQueryTerms={{view: "single", groupId: groupId}}
+            hideLegend={true}
+            mapOptions={{zoom:11, center: group.googleLocation.geometry.location, initialOpenWindows:[groupId]}}
+          />}
```

Comment:
To avoid having to figure out how to message different cases to the user, I decided to just go ahead and move the map to the bottom of the page, and make it smaller. So now all groups can add a banner image at the top.

<img width="600" alt="Screen Shot 2021-12-20 at 2 18 04 PM" src="https://user-images.githubusercontent.com/9057804/146821890-de5563da-29f5-4d58-bed5-f92f904cfd58.png">

Reply:
I think this makes the default view of a group worse, and most groups probably won't upload banner images. I'd say something like, if there's a banner image, put this down here, otherwise, put it at the top.

Reply:
Yeah I wasn't sure which was best here - ended up moving the map down for consistency between groups. But I take your point so I updated the code.